pages/common/gradle.md

```diff
@@ -11,15 +11,15 @@
 
 `gradle build -x {{test}}`
 
-- Run in offline mode to prevent gradle from accessing the network during builds:
+- Run in offline mode to prevent Gradle from accessing the network during builds:
 
 `gradle build --offline`
 
 - Clear the build directory:
 
 `gradle clean`
 
-- Compile and Release package:
+- Build an APK in release mode:
```

Comment:
```suggestion
- Build an APK package in release mode (specific to Android Studio):
```

Reply:
`APK` stands for `Android Application Package`, so `APK package` is redundant

Reply:
Thanks, I have edited my suggestion accordingly. :+1:

Reply:
Why not just use the abbreviation? Using the full form in the middle of the page might confuse some

Reply:
@nicokosi why do you think this is specific to Android Studio?

Reply:
Cf. my commit message:

> 'assembleDebug' and 'assembleRelease' tasks are specific to Android Studio:
> https://developer.android.com/studio/build/building-cmdline#ReleaseMode
> https://stackoverflow.com/questions/44185165/what-are-the-differences-between-gradle-assemble-and-gradle-build-tasks
> 
> They are not common tasks like 'build', 'assemble' and 'check':
> https://docs.gradle.org/current/userguide/command_line_interface.html#sec:command_line_executing_tasks

Reply:
@marchersimon : APK does mean nothing for me. IMHO, using the full name is explicit and searchable (even if it has some more characters).

Reply:
I thought the page was about buillding APKs, but since it just occurs once, I'm fine with using the long form

Reply:
```suggestion
- Build an Android Package (APK) in release mode:
```

Ref <https://android.stackexchange.com/a/13197/191699>

@nicokosi: `APK` is a pretty common abbreviation in the Android world.

I don't know enough about Gradle to know if this is specific to Android Studio or not.

Reply:
I don't get all the unnecessary detail, @ sbrl's suggestion works for me.

Reply:
I like the sbrl suggestion.